libparsec/crates/types/src/invite.rs

```diff
@@ -65,7 +64,6 @@ impl ToString for InvitationType {
 
 // SAS code is composed of 4 hexadecimal characters
 const SAS_CODE_CHARS: &[u8; 32] = b"ABCDEFGHJKLMNPQRSTUVWXYZ23456789";
```

Comment:
Should we add a comment explaining why we don't have `I` (uppercase `i`) and `1` ?

I though we had a comment like that but I don't remember where :thinking: maybe it was lost :shrug:

Reply:
I added a comment about that  :+1: